### ISSUE TITLE
Proposal: Remove space between function name and parameter declaration

### ISSUE THREAD

**Issue by robertrossmann** [strvcom/eslint-config-javascript#2]
## 1: Current enforced style:

``` js
function doStuff (param) {
// ...          ^
}
```
## 2: Proposed change:

``` js
function doStuff(param) {
// ...          ^
}
```

Which one do you prefer?
If you do not have a preference, please at least write "No preference" to this issue. 😀

> This proposal was raised by @jzavisek.

**Comment by vitbokisch** [comment]
I prefer the 2nd option ;)

**Comment by mdrbohlav** [comment]
2nd

**Comment by dannytce** [comment]
2nd

**Comment by robertrossmann** [comment]
This rule has been changed to the 2nd variant.

As a side-effect previously not thought of, the following will now be also encouraged:

``` js
// Anonymous functions
const fn = function() { /* ... */ }

// Anonymous generators
const gen = function*() { /* ... */ }
```

This change is [automatically fixable](http://eslint.org/docs/rules/space-before-function-paren) by ESLint.
